src/mjlab/scripts/demo.py

```diff
@@ -29,10 +29,9 @@ def main() -> None:
       motion_file=motion_path,
       num_envs=8,
       viewer="viser",
-      motion_command_sampling_mode="uniform",
     ),
   )
-  run_play("Mjlab-Tracking-Flat-Unitree-G1", args)
+  run_play("Mjlab-Tracking-Flat-Unitree-G1-Demo", args)
```

Comment:
Since the demo is a one-off thing and the only difference is sampling mode, I still kind of prefer just loading the play config and then overwriting the sampling mode